tests/test_transfers.py

```diff
@@ -0,0 +1,37 @@
+from audioop import add
+from subscrape.db.subscrape_db import SubscrapeDB
+import logging
+import subscrape
+import pytest
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("api", ["SubscanV2"])
+async def test_transfers(api):
+    
+    address = "DGaHm71gRHJGQVF2ubunkx37qgLpno2gYibq2WHoDJpFMt6"
+
+    config = {
+        "kusama":{
+            "_api": api,
+            "transfers":{
+                address : address
+            }
+        }
+    }
+    
+    logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(name)s - %(message)s')
+
+    logging.info("wiping storage")
+    subscrape.wipe_storage()
+    logging.info("scraping")
+    await subscrape.scrape(config)
+    logging.info("transforming")
+
+    db = SubscrapeDB("kusama")
+    transfers_storage = db.storage_manager_for_transfers(address)
+    transfers = dict(transfers_storage.get_iter())
+
+    first_transfer = transfers["7849231-5"]
```

Comment:
maybe first assert that that extrinsic was even found.

Reply:
my thinking is that it throws an exception either way, so why bother? or do you mean to provide more specific guidance to the developer?